tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2-ade20k/default/lite/1.md

```diff
@@ -2,19 +2,25 @@
 Lightweight deep learning model for semantic image segmentation.
 
 <!-- parent-model: sayakpaul/deeplabv3-mobilenetv2-ade20k/1 -->
-<!-- asset-path: https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/releases/download/v0.1.0/deeplabv3_mnv2_ade20k_train_2018_12_03_tflite.tar.gz -->
+<!-- asset-path: https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/releases/download/v0.8.0/mnv2_ade20k_tflite.tar.gz -->
```

Comment:
Assets on TF Hub are immutable so we cannot change the asset path. Instead we need to create a second version of the model. 

To do so you have to create a 2.md in this directory with the new MD changes and update the first line to:

`# Lite sayakpaul/deeplabv3-mobilenetv2-ade20k/1/default/2`

Reply:
Pushed the changes @maringeo